### DISCUSSION TITLE
默认用户名：admin 默认密码：streamx.报错No permission, please contact the administrator

### DISCUSSION THREAD

**Discussion by pzfok** [apache/streampark#2954] [General]
默认用户名：admin 默认密码：streamx.报错No permission, please contact the administrator

**Comment by caicancai** [comment]
<img width="526" alt="屏幕截图 2023-08-22 094343" src="https://github.com/apache/incubator-streampark/assets/77189278/07b5022e-18a0-4915-b468-61e191627492">
Try this
